libmachine/ssh/client.go

```diff
@@ -50,6 +50,8 @@ var (
 	baseSSHArgs = []string{
 		"-o", "PasswordAuthentication=no",
 		"-o", "StrictHostKeyChecking=no",
+		"-o", "ControlMaster=no",
```

Comment:
Isn't `ControlMaster=no` already set on line 59 and `ControlPath=no`?

Probably it just needs `ControlPath` line updated to `ControlPath=none`?